lib/utils/compose.js

```diff
@@ -17,90 +17,7 @@
 
 import * as path from 'path';
 import { ExpectedError } from '../errors';
-import { getChalk, stripIndent } from './lazy';
-
-export function appendOptions(opts) {
-	return opts.concat([
```

Comment:
I gather this code is no longer used after the conversion of the `build` and `deploy` commands to oclif.